dash/backend/scripts/proxy-webhook/auto-setup.sh

```diff
@@ -55,16 +58,22 @@ cp initialize_proxy.yaml init.yaml
 cp openssl.cnf ssl.cnf
 
 # Change the namespace in the initialize file and openssl configuration
-sed -I '' -e 's/m9sweeper-system/'$namespace'/g' init.yaml
-sed -I '' -e 's/m9sweeper-system/'$namespace'/g' ssl.cnf
+platform=$(uname)
+if [[ "$platform" != 'Linux' ]]; then
+    sed -i "s/m9sweeper-system/$namespace/g" init.yaml
```

Comment:
I noticed this is now lower case. Not sure if that was intended or not, otherwise I'll merge it.

Reply:
That is correct, `-I` with the capital I is not a valid argument in linux and in the help command for macOS it even shows to use a lowercase i.